### PR TITLE
tests: Add docker logs test

### LINKED ISSUE #244: add docker logs test
add functional docker logs test

### PR DESCRIPTION
**Pull request by GabyCT**
Add integration docker logs test

Fixes #244

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

**Comment by GabyCT** [comment]
setup issue #250

File: integration/docker/logs_test.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+	runCommand := func(args []string, expectedExitCode int) {
+		command = NewCommand(Docker, args...)
+		Expect(command).ToNot(BeNil())
+		exitCode = command.Run()
+		Expect(exitCode).To(Equal(expectedExitCode))
+	}
+
+	BeforeEach(func() {
+		id = RandID(30)
+		args = []string{"run", "-td", "--name", id, Image, "sh", "-c", "echo hello"}
```
Comment (grahamwhaley) [inline integration/docker/logs_test.go]: Do we check that we are getting `hello` back in the logs? I can't see that being done, and I think that is what we will get back from this echo, is it?

File: integration/docker/logs_test.go [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+	})
+
+	AfterEach(func() {
+		Expect(ContainerRemove(id)).To(BeTrue())
+		Expect(ContainerExists(id)).NotTo(BeTrue())
+	})
+
+	Describe("logs with docker", func() {
+		Context("check logs functionality", func() {
+			It("should work", func() {
```
Comment (grahamwhaley) [inline integration/docker/logs_test.go]: My instinct is that we should also have a test for something that does not work. The only one I can really think of is to ask for the logs from a container that does not exist, but then that is probably handled in docker itself without invoking any runtime parts - so, maybe that would not really be much use or addition here. Open for other ideas though.

**Comment by devimc** [comment]
lgtm

**Pull request merged**